toolkit/tools/pkg/imagecustomizerlib/liveosisobuilder.go

```diff
@@ -27,12 +27,24 @@ import (
 )
 
 const (
-	bootx64Binary         = "bootx64.efi"
-	grubx64Binary         = "grubx64.efi"
-	grubx64NoPrefixBinary = "grubx64-noprefix.efi"
+	rawBootloaderDir = "/boot/efi/EFI/BOOT"
```

Comment:
What does "raw" mean? Perhaps rename to `osEspBootloaderDir`?

Reply:
It applies to vhdx/qcow/raw images... picked raw as the most generic.
sure, osEsp prefix is better...